stage1/tasks/hangman/README.md

```diff
@@ -55,34 +69,38 @@ At the [following link](https://www.figma.com/file/ug2NAUiXPpaFDvch5TWUxd/Hangma
 - The pull request description must be provided according to the [template](https://docs.rs.school/#/pull-request-review-process?id=Требования-к-pull-request-pr).
 
 ## Technical requirements
+
 - The application is displayed correctly and functions properly in the latest version of Google Chrome.
 - It is **not allowed** to use:
-    - CSS frameworks (e.g., `Bootstrap`),
-    - JS frameworks (e.g., `Angular`, `React`, `Vue`, etc.),
-    - Outdated libraries (e.g. `JQuery`, etc.),
-    - `TypeScript`.
+  - CSS frameworks (e.g., `Bootstrap`),
+  - JS frameworks (e.g., `Angular`, `React`, `Vue`, etc.),
+  - Outdated libraries (e.g. `JQuery`, etc.),
+  - `TypeScript`.
 - You can use CSS preprocessors (`SASS`, `SCSS`) and `normalize.css`.
 - Using JS frameworks (e.g., Angular, React, Vue, etc.) is not allowed.
 
 ## CrossCheck Criteria (150 points)
-*It is recommended to print the secret word in the browser's console to facilitate the cross-check process.*
+
+_It is recommended to print the secret word in the browser's console to facilitate the cross-check process._
+
 - [ ] Responsive/adaptive UI from 1440px to 360px viewport: `+10`
 - [ ] The generation of DOM elements is implemented. `body` in the `index.html` is empty (can contain only `script` tag): `+20`
 - [ ] The game starts with the correct default view (empty gallows, underscores for secret word, etc.) and a random question: `+5`
 - [ ] The user can play the game by using the virtual keyboard: `+20`
 - [ ] The user can play the game by using the physical keyboard: `+20`
 - [ ] When the letter is correct, it appears instead of the corresponding blank line. If the letter repeats in the word, all corresponding underscores must be replaced by it: `+15`
```

Comment:
_Sorry, I just noticed that I'd missed one more "blank line"_

**Corrected:** - [ ] When the letter is correct, it appears instead of the corresponding underscore. If the letter repeats in the word, all the corresponding underscores must be replaced by it: `+15`

Reply:
done